osbs/exceptions.py

```diff
@@ -75,6 +75,10 @@ class OsbsAuthException(OsbsException):
     pass
 
 
+class OsbsCommitNotFound(OsbsException):
```

Comment:
I was thinking of having this defined only in module where it's used (not general OSBS exception)

Reply:
@MartinBasti I need make a correct checking by tests this part of code, so I must share this in test_utils.py

Reply:
Oh, I suggested bad idea. Understood and resolved :)